tools/dep_updaters/update-postject.sh

```diff
@@ -7,16 +7,26 @@
 
 set -ex
 
+ROOT=$(cd "$(dirname "$0")/../.." && pwd)
+[ -z "$NODE" ] && NODE="$ROOT/out/Release/node"
+[ -x "$NODE" ] || NODE=$(command -v node)
+NPM="$ROOT/deps/npm/bin/npm-cli.js"
+
+NEW_VERSION=$("$NODE" "$NPM" view postject dist-tags.latest)
+CURRENT_VERSION=$("$NODE" -p "require('./test/fixtures/postject-copy/node_modules/postject/package.json').version")
+
+if [ "$NEW_VERSION" = "$CURRENT_VERSION" ]; then
+  echo "Skipped because Postject is on the latest version."
+  exit 0
+fi
+
+echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
```

Comment:
I think it's a bit weird to expect `$GITHUB_ENV` to exist, it should either be output to stdout, or we should first check if the file exists.

Reply:
It's a mistake ... Let me remove it :/